.gitignore

```diff
@@ -23,4 +23,4 @@ target/
 .vscode
 .idea
 *.iml
-.tmp
+.tmp
```

Comment:
I can't seem to remove this.

Reply:
`.tmp` will be removed when we bring in the 2.0 changes.